src/main/target/CRAZYBEEF3FR/target.h

```diff
@@ -26,6 +26,9 @@
 #elif defined(CRAZYBEEF3DX)
 #define TARGET_BOARD_IDENTIFIER "CBDX"
 #define USBD_PRODUCT_STRING     "CrazyBee F3 DX"
+#elif defined(CRAZYBEEF3DXSPI)
```

Comment:
If we add this target as it is, owners of existing 'CrazyBee F3 DX' boards will suddenly have to choose between CRAZYBEEF3DX and CRAZYBEEF3DXSPI when selecting a target for a firmware update in configurator. I suspect a reasonable number of them will choose the wrong one, and as a result open a support request against Betaflight because their board does not work any more.
Also, this is not (yet) a target for which there is hardware for end users available commercially, which is normally a requirement for a target to be added to Betaflight. Because there are no other targets currently supporting the Spektrum SPI RX hardware, we can make an exception to the 'hardware available commercially' rule, but it should not be a target that we know will cause unnecessary support requests from users.

So my suggestions are:
- for now, submit a pull request to add the Spektrum SPI driver to a different target, where there is no potential for confusion with another target preconfigured for a Spektrum RX. Contact me on Slack (registration on https://slack.betaflight.com/) to work out details;
- when @githubDLG has finished design and testing for the Spektrum SPI version of the CRAZYBEEF3 board, choose a name for it that isn't another target's name with a suffix (maybe CRAZYBEEF3SP?).

Reply:
@mikeller I’ll be interested how this gets done in the end due to the similarity to my NOX with cc2500 “target” which your turned down as one of many possible modifications of an existing target.

Reply:
https://github.com/betaflight/betaflight/pull/7262

Reply:
Sorry, I wasn't aware that you were going to produce a board based on your design and start selling it, like @githubDLG is. Please add a board description  including links to where interested users can buy your board to your pull request.

Reply:
Except you proposed exactly this course of action before you knew there was a commercial board available. 


“Probably a good idea to add this to a different target instead:

there is already a CRAZYBEEF3 target with a Spektrum RX - having 2 of them to choose from will be confusing for users;
adding this to a target that has SPI pins exposed on a header (OMNIBUSF4FW, BEEROTORF4, MATEKFxx?, ...) will make it easier for other users to add their homebrew SPI RX mods.
You might want to talk to @githubDLG though, he might be interested in producing a board with a Spektrum SPI RX.”

Reply:
@joelucid:
- I don't think I owe you any sort of accountability, outside of maybe the pull requests that you open;
- I do not appreciate your attitude;
- the fundamental difference is that there currently is no target that includes   the Spektrum SPI driver. This means that the code is not verified as part of any CI build, and as such is at high risk of rotting if left this way - adding it to an arbitrary target (we could also use one of the DISCOVERY / NUCLEO targets) will mitigate this risk, until there is at least one target for a commercially available board using this target.

Reply:
Sorry, from your previous response I thought it was the lack of commercial availability you deemed decisive. You may not think you owe me anything but I believe it’s important for an open source community to have a deterministic decision process that can stand up to scrutiny. I can understand the new rationale you’ve provided. I don’t think scrutinizing the process should be taken as “bad attitude”. It’s certainly not meant this way. I mean this is not an absolute kingdom is it?

Reply:
No it's not an absolute kingdom. It's also not a commercial environment where somebody gets paid to maintain a library of documentation for all processes, so a little bit of trust into the maintainers to try and fairly apply best practices even in the absence of documentation for everything they do is called for (I think).
As stated before, this is an exception, for the reason stated before. The 'have  maximum CI coverage' trumps the 'only add targets that users can buy' rule in this case.